app/api/tickets.py

```diff
@@ -74,6 +79,20 @@ def before_create_object(self, data, view_kwargs):
 
 
 class TicketDetail(ResourceDetail):
+
+    def before_get_object(self, view_kwargs):
+        if view_kwargs.get('attendee_id') is not None:
+            try:
+                session = self.session.query(TicketHolder).filter_by(id=view_kwargs['attendee_id']).one()
+            except NoResultFound:
+                raise ObjectNotFound({'parameter': 'attendee_id'},
+                                     "Attendee: {} not found".format(view_kwargs['attendee_id']))
+            else:
+                if session.ticket_id is not None:
+                    view_kwargs['id'] = session.ticket_id
+                else:
+                    view_kwargs['id'] = None
+
     decorators = (jwt_required,)
     schema = TicketSchema
     data_layer = {'session': db.session,
```

Comment:
@poush
You forgot to add methods in **data_layer**.
Added it in [PR#3857](https://github.com/fossasia/open-event-orga-server/pull/3857)